### PR TITLE
Add tech learning page

### PR DESCRIPTION
**Pull request by erikdstock**
This is meant to be the start of a single README page for Tutorials We Love- an answer to the question 'Hey, I'm looking at picking up some work in [new language/paradigm/domain]- where should I start?'  I chose headers focused on languages and practices, with links in between them. I've compiled some that I picked up from looking around and will add more, but requested changes to this PR itself are also welcome!

### CODE REVIEW

**Comment by orta** [comment]
This is great

**Comment by eessex** [comment]
Maybe worth migrating the links from Notion into this

**Comment by erikdstock** [comment]
@eessex I started to!

File: recommendations/tech_learning.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+This is a living document; if you find something out of date or have an addition please open a [pull request](#).
+
+## Good starting points & tools
+
+| Name | Description |
+| -- | -- |
+| [b0rk's zines](https://jvns.ca/zines) | [Julia Evans](https://www.twitter.com/b0rk) makes great comics about *nix programming- [here's a team favorite on learning](https://jvns.ca/wizard-zine.pdf) |
+| [Learn X in Y minutes](https://learnxinyminutes.com/) | Scenic tours of pretty much every language |
+| [Symbol Hound](https://www.symbolhound.com/) | Symbol-aware searching ( e.g. `[ ? { > &` ) |
+
```
Comment (ashfurrow) [inline recommendations/tech_learning.md]: Julia Evans' work is amazing! Glitch also has a great zine: https://glitch.com/you-got-this

**Comment by orta** [comment]
Yep, this is a good doc - we should merge and iterate. Nice job @erikdstock

File: recommendations/tech_learning.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
+# Tech Learning
+We often hear that learning to google (or perhaps [DuckDuckGo](www.duckduckgo.com)) and use stack overflow is are the most important core competencies of any programmer, but growth and learning happen best outside the vacuum of the internet. This page is a list of the engineering team's favorite resources -tutorials, essays, starting points and general thoughts- on a variety of programming topics.
```
Comment (alloy) [inline recommendations/tech_learning.md]: > is ~~are~~

File: recommendations/tech_learning.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+| -- | -- |
+| [b0rk's zines](https://jvns.ca/zines) | [Julia Evans](https://www.twitter.com/b0rk) makes great comics about *nix programming- [here's a team favorite on learning](https://jvns.ca/wizard-zine.pdf) |
+| [Learn X in Y minutes](https://learnxinyminutes.com/) | Scenic tours of pretty much every language |
+| [exercism.io](https://www.exercism.io) | Algorithmic code challenges with social feedback |
+| [@damassi/learn-vim](https://github.com/damassi/learn-vim) | "It's not so bad, really." |
+| []() |  |
+
+&nbsp;
+-----
+## Languages + General Tools
```
Comment (alloy) [inline recommendations/tech_learning.md]: All of the tables in this section don’t appear to render as such.

File: recommendations/tech_learning.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+| [TypeScript Deep Dive](https://basarat.gitbooks.io/typescript/) | Get to the part where TypeScript feels good quicker |
+| []() |  |
+
+### Frontend iOS Practice
+*See also [Javascript](#javascript-+-nodejs) & [Swift + Objective C](#swift-+-objective-c)*
+| Name | Description |
+| -- | -- |
+| [React Native](https://facebook.github.io/react-native/) | The official docs |
+| [Expo](https://expo.io/) | 'a free and open source toolchain built around React Native to help you build native iOS and Android projects using JavaScript and React' |
+| []() |  |
```
Comment (alloy) [inline recommendations/tech_learning.md]: This table also doesn’t render.

**Comment by orta** [comment]
Yes! Get his shipped 🎉

**Comment by erikdstock** [comment]
I'll ask people to submit more resources tomorrow. Thanks and fine to merge by me- just commented out a big h1 question I'd left at the bottom.

**Comment by alloy** [comment]
Let’s do this! 👏

**Pull request merged**